src/app/components/grid/index.html

```diff
@@ -8,49 +8,91 @@
   </sky-demo-page-summary>
 
   <sky-demo-page-properties sectionHeading="Grid component properties">
+    <sky-demo-page-property
+      propertyName="columns"
+    >
+    Specifies the columns for the grid adn the column properties. This property accepts an array of <stache-code>SkyGridColumnModel</stache-code> values.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="data"
     >
-    Specifies an array of data for the grid. Each item in the array should have an ID and a property that maps to either the <stache-code>field</stache-code> or <stache-code>id</stache-code> property of each column in the grid.
+    Specifies the data for the grid. Each item requires an ID and a property that maps to the <stache-code>field</stache-code> or <stache-code>id</stache-code> property of each column in the grid. This property accepts an array of <stache-code>any</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="selectedColumnIds"
+      propertyName="enableMultiselect"
+      defaultValue="false"
       isOptional="true"
     >
-    Specifies an array of columns to display in the grid. Will match the <stache-code>id</stache-code> or <stache-code>field</stache-code> properties of the columns. If no array is provided, then the grid displays all columns. Acceptable values: <stache-code>Array of strings</stache-code>.
+    Indicates whether to enable the multiselect feature that displays a column of checkboxes on the left side of the grid. To display the checkboxes, you must also specify the <stache-code>multiselectRowId</stache-code> property. This property accepts <stache-code>boolean</stache-code> values.
```

Comment:
`To display the checkboxes, you must also specify the <stache-code>multiselectRowId</stache-code> property.`
This is no longer true. If you don't provide the `multiselectRowId` property, then it defaults to the `id` property of the provided data.